little_raft/src/replica.rs

```diff
@@ -396,8 +398,25 @@ where
                     self.match_index.insert(from_id, last_index);
                 } else {
                     // Update information about the peer's logs.
-                    self.next_index
-                        .insert(from_id, self.next_index[&from_id] - 1);
+                    //
+                    // If the mismatch_index is no less than the existing
```

Comment:
s/no less/is greater than equal to/ (negatives are hard to read)

Reply:
Fixed